SoftwareConstructionFundamental/SCF-Session8/zoo/Zoo.java

```diff
@@ -0,0 +1,247 @@
+package zoo;
+
+import zone.*;
+import cage.*;
+import animal.*;
+
+import java.util.List;
+import java.time.zone.ZoneRulesException;
+import java.util.ArrayList;
+
+/**
+ * It will have all properties and operations of Zoo
+ * 
+ * @author Shubham Jain
+ *
+ */
+public class Zoo {
+	public static List<Zone> zoneList = new ArrayList<Zone>();
+	public static List<Cage> cageList = new ArrayList<Cage>();
+	public static List<Animal> animalList = new ArrayList<Animal>();
+	static int zoneId = 1;
+	static int cageId = 1;
+	static int animalId = 1;
+
+	public Zoo() {
+		init();
+	}
+
+	public void init() {
+
+		zoneList.clear();
+		animalList.clear();
+		cageList.clear();
+
+		addZone(2, "Mammal", true, false);
```

Comment:
@meta-shubham-jain 
Remove initialisation from class constructor. Create an object in JUnit class and add one by one.

Reply:
done